core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,248 @@
+package edu.asu.c3simulator.simulation;
+
+import java.util.Random;
+
+public class Employee
+{
+	public String employeeName;
+	public String employeePosition;
+	public String randomEmployeePosition;
+	public int employeePay;
+	public int employeePref;
+	public int employeeMorale;
```

Comment:
These names seem very smurfy... 
Code will look like this: employee.getEmployeeMoral() 
which seems repetitive

Consider renaming